### PR TITLE
For Content Review - GSQ Dataset Themes

### PR DESCRIPTION
**Pull request by KellyVance**
https://vocabs.test.gsq.digital/vocabulary/gsq-dataset-themes
Custodian - @mark-gordon-dnrme 
Peer Reviewer 1 - @johnmckellar 
Peer Reviewer 2 - @LizDerrington

### CODE REVIEW

File: vocabularies/gsq-dataset-themes.ttl [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    skos:prefLabel "Activities and Results"@en .
+
+:ancillary-information a skos:Collection ;
+    skos:definition "Data descriptive of the organisations involved in data collection and the administrative constraints and associations under which data is collected or related."@en ;
+    skos:member :administrative-data,
+        :spatial-data ;
+    skos:prefLabel "Ancillary Information"@en .
+
+:products-and-documents a skos:Collection ;
+    skos:definition "The final products and official reports that synthesis the collected data into documentation that describe activities and/or provide interpretive and contextual information."@en ;
```
Comment (johnmckellar) [inline vocabularies/gsq-dataset-themes.ttl]: reports that synthesise … <<< Forget, as I now see that this has been done.

File: vocabularies/gsq-dataset-themes.ttl [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:publications a skos:Concept ;
+    skos:definition "Publications that describe the geology, resource potential, or other areas of departmental interest such as books, journals, periodicals, presentations, posters, academic publications, or multimedia products."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Publications"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:reports a skos:Concept ;
+    skos:definition "Reports lodged as part of an approved study or activity, commonly for the purpose of compliance with permit grant and retention conditions. These reports may be permit based such as a resource authority annual report, activity based such as well completion reports, or non-tenure reports such as departmental survey reports."@en ;
```
Comment (LizDerrington) [inline vocabularies/gsq-dataset-themes.ttl]: Minor change to:-"Reports lodged as part of an approved study or activity, commonly for the purpose of compliance with permit grant and retention conditions. These reports may be permit based such as resource authority annual reports, activity based such as well completion reports, or non-tenure reports such as departmental survey reports or academic studies.

File: vocabularies/gsq-dataset-themes.ttl [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Reports"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:resource-statistics a skos:Concept ;
+    skos:definition "Statistical data and reports submitted to the department or collected/generated by the department, particularly production and sales quantities."@en ;
```
Comment (LizDerrington) [inline vocabularies/gsq-dataset-themes.ttl]: Minor change to 'Statistical data and reports submitted to the department or collated/generated by the department, particularly production, sales and reserve quantities'

File: vocabularies/gsq-dataset-themes.ttl [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Resource statistics"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:spatial-data a skos:Concept ;
+    skos:definition "Data and metadata related to spatial objects and representations such as maps, aerial and satellite imagery, spatial data files and databases."@en ;
```
Comment (LizDerrington) [inline vocabularies/gsq-dataset-themes.ttl]: Change to 'Spatial data files and databases including data and metadata related to spatial objects and representations such as maps, aerial and satellite imagery'

File: vocabularies/gsq-dataset-themes.ttl [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:activities-and-results a skos:Collection ;
+    skos:definition "Primary dataset that describe the geological features of interest and the activities that generate those datasets."@en ;
```
Comment (LizDerrington) [inline vocabularies/gsq-dataset-themes.ttl]: Maybe "Primary dataset that describe the geological features of interest and the activities that generate the results datasets."

File: vocabularies/gsq-dataset-themes.ttl [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    skos:prefLabel "Activities and results"@en .
+
+:ancillary-information a skos:Collection ;
+    skos:definition "Data descriptive of the organisations involved in data collection and the administrative constraints and associations under which data is collected or related."@en ;
+    skos:member :administrative-data,
+        :spatial-data ;
+    skos:prefLabel "Ancillary information"@en .
+
+:products-and-documents a skos:Collection ;
+    skos:definition "The final products and official reports that synthesise the collected data into documentation that describe activities and/or provide interpretive and contextual information."@en ;
```
Comment (LizDerrington) [inline vocabularies/gsq-dataset-themes.ttl]: Maybe "The final products and official reports that synthesise the collected data into documentation that describe activities, results and/or provide interpretive and contextual information"

File: vocabularies/gsq-dataset-themes.ttl [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ dataset themes"@en .
+
+:administrative-data a skos:Concept ;
+    skos:definition "Ancillary information regarding the administration of Queensland Resources such as adminstrative boundaries, mining permits, legislation, and information about organisations such as DNRME and GSQ."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Mining Permit vs Resource Authority is a contentious issue. But have added resource authorities into definition.

File: vocabularies/gsq-dataset-themes.ttl [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Administrative data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-data a skos:Concept ;
+    skos:definition "Primary descriptive data defining the geological attributes of a feature of interest."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Changed to simplified definition "Data and metadata that define and describe the geological attributes of a feature of interest."

File: vocabularies/gsq-dataset-themes.ttl [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Geoscience data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:geoscience-project-data a skos:Concept ;
+    skos:definition "Data and media describing project information such as activities performed, planning and approvals for the collection and analysis of primary geoscience data."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Changed wording

File: vocabularies/gsq-dataset-themes.ttl [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+    skos:topConceptOf :conceptScheme .
+
+:other a skos:Concept ;
+    skos:definition "Any other metadata, data, or information that is not described by any of the other GSQ Dataset Themes."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Other"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:operational-data a skos:Concept ;
+    skos:definition "Data describing the operations of mining, drilling, processing, and sales operations e.g. borehole drilling, pipelines, processing facilities."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed

File: vocabularies/gsq-dataset-themes.ttl [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Operational data"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:physical-samples a skos:Concept ;
+    skos:definition "Data and metadata pertaining to physical samples taken during geoscience data acquisition e.g. drill core, rock chips, rock samples from outcrop, soil, sediment, water, drilling fluids, oil, gas etc."@en ;
```
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed.

File: vocabularies/gsq-dataset-themes.ttl (hunk 2 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed.

File: vocabularies/gsq-dataset-themes.ttl (hunk 3 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed.

File: vocabularies/gsq-dataset-themes.ttl (hunk 4 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Definition rewritten for greater clarity.

File: vocabularies/gsq-dataset-themes.ttl (hunk 5 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed.

File: vocabularies/gsq-dataset-themes.ttl (hunk 6 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Sentence changed.

File: vocabularies/gsq-dataset-themes.ttl (hunk 7 above)
[the remark this answers is not in the document]
Comment (LizDerrington) [reply vocabularies/gsq-dataset-themes.ttl]: @DavidCrosswellGSQ @mark-gordon-dnrme @GSQ-AI  Why 'mining permit'? Please comment as we really dislike this terminology as it is misleading and inaccurate.

[the remark this answers is not in the document]
Comment (GSQ-AI) [reply vocabularies/gsq-dataset-themes.ttl]: I agree with Liz, I really do not feel "Mining" permit is appropriate, especially when we're referring to permits where mining does not occur. This is definitely a broader issue though.

**Comment by mark-gordon-dnrme** [comment]
I don't think we should have a separate spatial data type IMHO?

File: vocabularies/gsq-dataset-themes.ttl [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+    skos:definition "Statistical data and reports submitted to the department or generated by the department, particularly production, sales and reserve quantities."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Resource statistics"@en ;
+    skos:topConceptOf :conceptScheme .
+
+:spatial-data a skos:Concept ;
+    skos:definition "Data and metadata describing spatial properties and features, including maps, aerial and satellite imagery, and files for use in GIS software."@en ;
+    skos:inScheme :conceptScheme ;
+    skos:prefLabel "Spatial data"@en ;
+    skos:topConceptOf :conceptScheme .
```
Comment (KellyVance) [inline vocabularies/gsq-dataset-themes.ttl]: As discussed throughout the data modelling and scoping of system requirements  all geoscience data is spatial in some sense as is all the supporting information that goes with it. Therefore is Spatial Data a redundant category? Virtually all our information would have a spatial aspect.

Comment (dxwell) [reply vocabularies/gsq-dataset-themes.ttl]: I think there is a difference between a spatial dataset and a dataset that has spatial attributes. Where would you put a topographic map?

File: vocabularies/gsq-dataset-themes.ttl (hunk 7 above)
[the remark this answers is not in the document]
Comment (dxwell) [reply vocabularies/gsq-dataset-themes.ttl]: From https://en.wikipedia.org/wiki/Mining

Mining is the extraction of valuable minerals or other geological materials from the Earth, usually from an ore body, lode, vein, seam, reef or placer deposit. These deposits form a mineralized package that is of economic interest to the miner.

Ores recovered by mining include metals, coal, oil shale, gemstones, limestone, chalk, dimension stone, rock salt, potash, gravel, and clay. Mining is required to obtain any material that cannot be grown through agricultural processes, or feasibly created artificially in a laboratory or factory. Mining in a wider sense includes extraction of any non-renewable resource such as petroleum, natural gas, or even water.

[the remark this answers is not in the document]
Comment (dxwell) [reply vocabularies/gsq-dataset-themes.ttl]: Department of Natural Resources, **Mines** and Energy

File: vocabularies/gsq-dataset-themes.ttl (hunk 12 above)
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: Topo maps would be under geoscience data. But the same rationale would have to apply to aerial and satellite imagery... which may or may not apply to geoscience in anyway. I'd be dubious labelling it as such.
Hypothetically if we had some environmental data that was not strictly geoscience data it wouldn't fit neatly into any of the categories either. So perhaps there is room for another category for the scientific data that is _not_ strictly geoscience data.

@mark-gordon-dnrme thoughts?

File: vocabularies/gsq-dataset-themes.ttl (hunk 7 above)
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: I'm in favour of a terminology change. The problem is it is a terrible definition and the only way we can contribute to its rectification is to not perpetuate its misapplied usage. In common industry, academic, and professional usage one does _not_ mine petroleum or water. It has also had negative socio-political implications in the past.

DNRME also covers land usage and valuations, indigenous land use, vegetation management, water and sewerage, surveying, place names, and pyrotechnic displays, so some abbreviation and omission in the Department title is expected and not a reason to insist on continuing to misapply other definitions.

The wiki quote doesn't have a primary source but is quoted on numerous websites (non-of-which reference source material). 

Oxford dictionary Mining: "The process or industry of obtaining coal or other minerals from a mine." 
Calling a gas field a mine would be problematic at best. 

Britannica:  "the process of extracting minerals from the surface of the Earth, including the seas."
Hydrocarbons are not minerals. Water is not a mineral (except when it is ice)

Cambridge: "the industry or activity of removing substances such as coal or metal from the ground by digging "
Petroleum products. Not coal, not minerals, not metals, not metal-bearing minerals. boring and drilling =/= digging.

https://www.mindat.org/glossary/Mine of the defintions quoted only iii mentions oil or wells and explicitly _excludes_ them from being included in the definition of mining.

and to quote another wiki page
https://en.wikipedia.org/wiki/Extraction_of_petroleum
The extraction of petroleum is the process by which usable petroleum is drawn out from beneath the earth's surface location.

The exception (when hydrocarbon extraction can be accurately described as mining) is when you are mining oil sands or oil shales.

File: vocabularies/gsq-dataset-themes.ttl (hunk 12 above)
Comment (GSQ-AI) [reply vocabularies/gsq-dataset-themes.ttl]: The definition of Spatial Data as "Data and metadata describing spatial properties and features, including maps, aerial and satellite imagery, and files for use in GIS software." might be contributing to this ambiguity around the content.

The relevant items I can think of that would fit this current definition would be:

- Wellcards (lat/lon, depth, geometry), las files & other activity reports (Annual etc.)
- Digital Elevation Models
- Topographic Maps
- Tenure plans
- Mine Plans
- Radar traces
- Satellite Imagery
- Aerial Photography
- LIDAR
- JORC reports
- Journal articles
- Geophysics
etc.

What is the overall purpose for this category, what are we trying to capture in it?

Comment (DavidCrosswellGSQ) [reply vocabularies/gsq-dataset-themes.ttl]: Change spatial data definition to:
> Spatial data including maps, aerial and satellite imagery, and files for use in GIS software.

File: vocabularies/gsq-dataset-themes.ttl (hunk 7 above)
[the remark this answers is not in the document]
Comment (DavidCrosswellGSQ) [reply vocabularies/gsq-dataset-themes.ttl]: @KellyVance please change this.

[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: changed @LizDerrington

**Comment by KellyVance** [comment]
> Guys - I think we need to remove the spatial data category. It should not be a content type on it's own IMHO. Cheers, M

@mark-gordon-dnrme I have changed the definition to narrow the scope of this category. After discussion with the team we have agreed it is a valid category that may not always be captured elsewhere. Prepared to proceed and add/delete/edit categories as we begin actual categorisation of our data assets. I recommend approval for the first edition.

File: vocabularies/gsq-dataset-themes.ttl [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-09T15:16:00+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ dataset themes"@en .
+
+:administrative-data a skos:Concept ;
+    skos:definition "Ancillary information regarding the administration of Queensland Resources such as adminstrative boundaries, resource permits, legislation, and information about organisations such as DNRME and GSQ."@en ;
```
Comment (DavidCrosswellGSQ) [inline vocabularies/gsq-dataset-themes.ttl]: Typo "adminstrative"

Comment (KellyVance) [reply vocabularies/gsq-dataset-themes.ttl]: fixed

**Pull request merged**